public/js/combinationalAnalysis.js

```diff
@@ -31,16 +31,16 @@ createCombinationalAnalysisPrompt=function(scope=globalScope){
 }
 function createBooleanPrompt(inputListNames,outputListNames,scope=globalScope){
 
-    inputListNames=inputListNames||(prompt("Enter inputs separated by space").split(' '));
-    outputListNames=outputListNames||(prompt("Enter outputs separated by space").split(' '));
+    inputListNames=inputListNames||(prompt("Enter inputs separated by commas").split(','));
+    outputListNames=outputListNames||(prompt("Enter outputs separated by commas").split(','));
 
     var s='<table>';
     s+='<tbody style="display:block; max-height:70vh; overflow-y:scroll" >';
     s+='<tr>';
     for(var i=0;i<inputListNames.length;i++)
-        s+='<th>'+inputListNames[i]+'</th>';
+        s+='<th>'+inputListNames[i].trim()+'</th>';
     for(var i=0;i<outputListNames.length;i++)
-        s+='<th>'+outputListNames[i]+'</th>';
+        s+='<th>'+outputListNames[i].trim()+'</th>';
```

Comment:
Can you perform the trimming before itself on the actual list `inputListNames` and `outputListNames` because we are using the names in several places including the labels for the final input and output circuit elements!

Reply:
Oh I will get it done